docs/project-service-management-guide.md

```diff
@@ -72,11 +72,12 @@ This guide assumes a few things:
       [here](https://docs.microsoft.com/en-us/azure/devops/repos/git/create-new-repo?view=azure-devops#create-a-repo-using-the-web-portal)
 3. The application is packaged and run through a Docker image hosted on
    [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/)
-4. The user running `spk` has full access to the above resources.
+4. The user is running `spk` has full access to the above resources.
```

Comment:
this line was fine before.

Could be changed to
>The user that is running `spk` has full access to the above resources.

Reply:
done